tests/infra/network.py

```diff
@@ -403,21 +403,23 @@ def start_and_join(self, args):
                 "cp", args.js_app_script, args.binary_dir
             ).check_returncode()
             self.consortium.set_js_app(
```

Comment:
Could we do the same thing for the recovery sequence, in the `recover()` function below?

Reply:
Good idea, let's find out :)